test/parse.js

```diff
@@ -48,6 +48,13 @@ t.test('parse(text)', t => {
             'parses escaped property names'
         )
 
+        t.strictSame(
+            // eslint-disable-next-line no-proto
+            JSON5.parse('{"__proto__":1}').__proto__,
```

Comment:
Being able to specify __proto__ is the bug, why was a test written to assert that the bug happens?